contracts/oracle/integration/test-callback.ava.js

```diff
@@ -0,0 +1,67 @@
+import { Worker, NEAR } from "near-workspaces";
+import test from "ava";
+
+const claim_b64 = "EQAAAGNsYWltZXIudGVzdC5uZWFyBAAAADB4MWELAAAAAAAAAA==";
```

Comment:
please add a comment explaining where this comes from.

Reply:
DONE